include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -580,15 +610,29 @@ constexpr function_wrapper_t alignment_configurator::configure_scoring_scheme(co
 {
     using traits_t = alignment_configuration_traits<config_t>;
 
-    using alignment_scoring_scheme_t =
-        lazy_conditional_t<traits_t::is_vectorised,
-                           lazy<simd_match_mismatch_scoring_scheme,
-                                typename traits_t::score_type,
-                                typename traits_t::scoring_scheme_alphabet_type,
-                                typename std::conditional_t<traits_t::is_global,
-                                                            seqan3::align_cfg::method_global,
-                                                            seqan3::detail::method_local_tag>>,
-                            typename traits_t::scoring_scheme_type>;
+    using scoring_scheme_t = typename traits_t::scoring_scheme_type;
+    constexpr bool is_aminoacid_scheme = is_type_specialisation_of_v<scoring_scheme_t, aminoacid_scoring_scheme>;
+    using alignment_type_t = typename std::conditional_t<traits_t::is_global,
+                                                         seqan3::align_cfg::method_global,
+                                                         seqan3::detail::method_local_tag>;
+
+    using simple_simd_scheme_t = lazy_conditional_t<traits_t::is_vectorised,
```

Comment:
I guess the code duplication is because of the old and new alignment algorithm? otherwise you might want to refactor this into a type trait to avoid inconsitencies.

Reply:
yes, I was like "Gosh, do i really need to refactor that if I know it is going to be removed soon?" And I thought "... nah!"